posts/panel_material_ui_announcement/index.ipynb

```diff
@@ -0,0 +1,1927 @@
x1ZSI6NS40NzcyMjU1NzUwNTE2NjF9LCJsaW5lX2NvbG9yIjp7InR5cGUiOiJmaWVsZCIsImZpZWxkIjoiY29sb3IiLCJ0cmFuc2Zvcm0iOnsiaWQiOiJlOGU4NjBkOC1kMjAxLTQ2NzktYTRkOC0zMzRkMWU5N2IyZDIifX0sImxpbmVfYWxwaGEiOnsidHlwZSI6InZhbHVlIiwidmFsdWUiOjAuMX0sImZpbGxfY29sb3IiOnsidHlwZSI6ImZpZWxkIiwiZmllbGQiOiJjb2xvciIsInRyYW5zZm9ybSI6eyJpZCI6ImU4ZTg2MGQ4LWQyMDEtNDY3OS1hNGQ4LTMzNGQxZTk3YjJkMiJ9fSwiZmlsbF9hbHBoYSI6eyJ0eXBlIjoidmFsdWUiLCJ2YWx1ZSI6MC4xfSwiaGF0Y2hfY29sb3IiOnsidHlwZSI6ImZpZWxkIiwiZmllbGQiOiJjb2xvciIsInRyYW5zZm9ybSI6eyJpZCI6ImU4ZTg2MGQ4LWQyMDEtNDY3OS1hNGQ4LTMzNGQxZTk3YjJkMiJ9fSwiaGF0Y2hfYWxwaGEiOnsidHlwZSI6InZhbHVlIiwidmFsdWUiOjAuMX19fSwiaG92ZXJfZ2x5cGgiOnsidHlwZSI6Im9iamVjdCIsIm5hbWUiOiJTY2F0dGVyIiwiaWQiOiIxMTk0MjhiZi05NDRmLTQ4MWItODIyNC02YTA1NmFjMDFlZjEiLCJhdHRyaWJ1dGVzIjp7InRhZ3MiOlsiYXBwbHlfcmFuZ2VzIl0sIngiOnsidHlwZSI6ImZpZWxkIiwiZmllbGQiOiJiaWxsX2xlbmd0aF9tbSJ9LCJ5Ijp7InR5cGUiOiJmaWVsZCIsImZpZWxkIjoiYmlsbF9kZXB0aF9tbSJ9LCJzaXplIjp7InR5cGUiOiJ2YWx1ZSIsInZhbHVlIjo1LjQ3NzIyNTU3NTA1MTY2MX0sImxpbmVfY29sb3IiOnsidHlwZSI6ImZpZWxkIiwiZmllbGQiOiJjb2xvciIsInRyYW5zZm9ybSI6eyJpZCI6ImU4ZTg2MGQ4LWQyMDEtNDY3OS1hNGQ4LTMzNGQxZTk3YjJkMiJ9fSwiZmlsbF9jb2xvciI6eyJ0eXBlIjoiZmllbGQiLCJmaWVsZCI6ImNvbG9yIiwidHJhbnNmb3JtIjp7ImlkIjoiZThlODYwZDgtZDIwMS00Njc5LWE0ZDgtMzM0ZDFlOTdiMmQyIn19LCJoYXRjaF9jb2xvciI6eyJ0eXBlIjoiZmllbGQiLCJmaWVsZCI6ImNvbG9yIiwidHJhbnNmb3JtIjp7ImlkIjoiZThlODYwZDgtZDIwMS00Njc5LWE0ZDgtMzM0ZDFlOTdiMmQyIn19fX0sIm11dGVkX2dseXBoIjp7InR5cGUiOiJvYmplY3QiLCJuYW1lIjoiU2NhdHRlciIsImlkIjoiNGQ0MjRmMGUtYjFjOC00NzYxLWIwODQtY2YxYmFmMDllMWQxIiwiYXR0cmlidXRlcyI6eyJ0YWdzIjpbImFwcGx5X3JhbmdlcyJdLCJ4Ijp7InR5cGUiOiJmaWVsZCIsImZpZWxkIjoiYmlsbF9sZW5ndGhfbW0ifSwieSI6eyJ0eXBlIjoiZmllbGQiLCJmaWVsZCI6ImJpbGxfZGVwdGhfbW0ifSwic2l6ZSI6eyJ0eXBlIjoidmFsdWUiLCJ2YWx1ZSI6NS40NzcyMjU1NzUwNTE2NjF9LCJsaW5lX2NvbG9yIjp7InR5cGUiOiJmaWVsZCIsImZpZWxkIjoiY29sb3IiLCJ0cmFuc2Zvcm0iOnsiaWQiOiJlOGU4NjBkOC1kMjAxLTQ2NzktYTRkOC0zMzRkMWU5N2IyZDIifX0sImxpbmVfYWxwaGEiOnsidHlwZSI6InZhbHVlIiwidmFsdWUiOjAuMn0sImZpbGxfY29sb3IiOnsidHlwZSI6ImZpZWxkIiwiZmllbGQiOiJjb2xvciIsInRyYW5zZm9ybSI6eyJpZCI6ImU4ZTg2MGQ4LWQyMDEtNDY3OS1hNGQ4LTMzNGQxZTk3YjJkMiJ9fSwiZmlsbF9hbHBoYSI6eyJ0eXBlIjoidmFsdWUiLCJ2YWx1ZSI6MC4yfSwiaGF0Y2hfY29sb3IiOnsidHlwZSI6ImZpZWxkIiwiZmllbGQiOiJjb2xvciIsInRyYW5zZm9ybSI6eyJpZCI6ImU4ZTg2MGQ4LWQyMDEtNDY3OS1hNGQ4LTMzNGQxZTk3YjJkMiJ9fSwiaGF0Y2hfYWxwaGEiOnsidHlwZSI6InZhbHVlIiwidmFsdWUiOjAuMn19fX19XSwidG9vbGJhciI6eyJ0eXBlIjoib2JqZWN0IiwibmFtZSI6IlRvb2xiYXIiLCJpZCI6ImFmOGEzZDEwLTRlNzAtNDc5OS04YjZiLTgwMGE5ZmM3YjBjZCIsImF0dHJpYnV0ZXMiOnsidG9vbHMiOlt7InR5cGUiOiJvYmplY3QiLCJuYW1lIjoiV2hlZWxab29tVG9vbCIsImlkIjoiYjUxNzc2YjAtY2E5MS00YjBjLWE1NGMtY2U2Y2I0ODhhNzUyIiwiYXR0cmlidXRlcyI6eyJ0YWdzIjpbImh2X2NyZWF0ZWQiXSwicmVuZGVyZXJzIjoiYXV0byIsInpvb21fdG9nZXRoZXIiOiJub25lIn19LHsidHlwZSI6Im9iamVjdCIsIm5hbWUiOiJIb3ZlclRvb2wiLCJpZCI6IjMyNzc4ZDllLTE3NzYtNDZjYi04ZGU5LTFjMmI5MGU4Nzc2NCIsImF0dHJpYnV0ZXMiOnsidGFncyI6WyJodl9jcmVhdGVkIl0sInJlbmRlcmVycyI6W3siaWQiOiJlNzNjNWQ2Zi1kNzg5LTQwYjQtYWRlMC05YzY5MjliOTE2ZjQifV0sInRvb2x0aXBzIjpbWyJiaWxsX2xlbmd0aF9tbSIsIk
```

Comment:
I'm confused by what Github is doing here in this edit; all I want to do is to correct "toggle" to "toggles", but the diff comes out of nowhere and is enormous! Maybe don't apply this change and simply make that one-character edit manually.

```suggestion
    "Note how the switcher toggles between large and small implementations and the grid adjusts to the configured breakpoints."
```